clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultEventHandler.java

```diff
@@ -179,27 +147,6 @@ public boolean add(final ApplicationEvent event) {
         return applicationEventQueue.add(event);
     }
 
-    // bootstrap a metadata object with the bootstrap server IP address,
```

Comment:
Reviewer's note: moved this code to the constructor and it's now using the utility classes/methods for less verbosity.